test/test-docker-build.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Docker image build tests
+
+# If docker is not installed, skip the whole docker test
+# Travis CI does not currently support docker on OSX (https://docs.travis-ci.com/user/docker/)
+if ! hash docker 2>/dev/null; then
+  echo "Docker is not installed on this host"
+  exit 0
```

Comment:
Rather than exiting, can we skip instead so we can clearly see when docker isn't available in the tests.

Reply:
Sure, I didn't see a skip test function in the testlib.sh, is there are preferred way to do that?

Reply:
Oooops. I thought we'd included that function. 😊 Ignore me. This will do.

Reply:
It will output`"Docker is not installed on this host"` in the tests if there is no `docker` available as well: 

```
==> Running testsuite with GHE_TEST_REMOTE_VERSION=11.10.344
Docker is not installed on this host
...
```